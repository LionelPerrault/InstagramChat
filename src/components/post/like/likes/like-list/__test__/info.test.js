import React from 'react'
import { create } from 'react-test-renderer'
import LikeInfo from '../info'

describe('LikeInfo Component', () => {
  it('should match snapshot', () => {
    const tree = create(
      <LikeInfo
        likeDetails={{
          like_by: 7,
          username: 'ghalib',
          like_time: '1516528068337',
        }}
      />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
