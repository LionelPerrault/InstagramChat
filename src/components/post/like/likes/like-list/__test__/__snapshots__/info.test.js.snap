// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LikeInfo Component should match snapshot 1`] = `
<div
  className="modal_it_info"
>
  <Link
    className="modal_it_username"
    to="/profile/ghalib"
  >
    ghalib
  </Link>
  <span
    className="modal_it_light"
  >
    a month ago
  </span>
</div>
`;
